src/thank-you.hbs

```diff
@@ -0,0 +1,18 @@
+---
+title: Thank you!
+description: Thank you for supporting our continued work to develop products for student success
+---
+
+<section class="general section is-revealing">
+	<div class="general-inner section-inner">
+		<div class="container-sm">
+			<h1>Thank you!</h1>
+			<p>Thank you for supporting our continued work to develop products for student success.</p>
+			<div class="m-auto text-center">
+				<img class="m-auto pb-64 w-50p" src="/static/images/undraw_make_it_rain_iwk4.svg" />
+				<a class="m-auto button button-primary button-wide-mobile"
+					href="{{site.auth}}/api/v0/session/begin?gb-login">Return to Gradebook</a>
```

Comment:
We should use Update CTA for this, but also it should be `/api/v0/session/redirect` since that will always take you to the right place